sirepo/pkcli/service.py

```diff
@@ -135,7 +136,11 @@ def jupyterhub():
         f = d.join('conf.py')
         pkjinja.render_resource(
             'jupyterhub_conf.py',
-            PKDict(_cfg()).pkupdate(**sirepo.sim_api.jupyterhublogin.cfg),
+            PKDict(_cfg()).pkupdate(
+                # POSIT: Running with nginx and uwsgi
+                sirepo_uri=f'http://{socket.getfqdn()}:{_cfg().nginx_proxy_port}',
```

Comment:
Since this is only for development, this is ok. You might actually want to hardwire to 127.0.0.1?

Reply:
127.0.0.1 doesn't work because then `flask.url_for('_dispatch_empty', _external=external)` returns 127.0.0.1 which messes up the redirect to the login pages.